source/_integrations/telegram.markdown

```diff
@@ -18,7 +18,7 @@ The requirements are:
 - You need to configure a [Telegram bot in Home Assistant](/integrations/telegram_bot) and define there your API key and the allowed chat ids to interact with.
 - The `chat_id` of an allowed user.
 
-The quickest way to retrieve your `chat_id` is visiting [https://api.telegram.org/botYOUR_API_TOKEN/getUpdates](https://api.telegram.org/botYOUR_API_TOKEN/getUpdates) or to use `$ curl -X GET https://api.telegram.org/botYOUR_API_TOKEN/getUpdates` **after** you have sent the bot a message. Replace `YOUR_API_TOKEN` with your actual token.
+The quickest way to retrieve your `chat_id` is visiting `https://api.telegram.org/botYOUR_API_TOKEN/getUpdates` or to use `$ curl -X GET https://api.telegram.org/botYOUR_API_TOKEN/getUpdates` **after** you have sent the bot a message. Replace `YOUR_API_TOKEN` with your actual token.
```

Comment:
With that change it's no longer a link.

Reply:
I personally, agree on not making it a link. The link would be invalid and giving a 404, rather a user should be able to copy, paste and adjust this easy. Besides that, dead links, are never postive for your website.

Reply:
> With that change it's no longer a link.

Yeah, that was intentional. This is a broken link in which one should replace YOUR_API_TOKEN with some other information, hence a code block.

Reply:
The old way was straight-forward. Click the link, copy the token and add the token in the browser's address line. Now one needs to copy the link, paste it into the address field of the browser, copy the token and add it.

If you guys agree then I fine with it.